cloud_images/centos7/create_base_ami.sh

```diff
@@ -53,7 +53,9 @@ ONBOOT="yes"
 TYPE="Ethernet"
 USERCTL="yes"
 PEERDNS="yes"
-IPV6INIT="no"
+IPV6INIT="yes"
+DHCPV6C="yes"
+NM_CONTROLLED="no"
```

Comment:
I am just curious, do not know the context. NM got really good, basing on my experience and `nmcli` rocks.